qutebrowser/utils/log.py

```diff
@@ -505,7 +505,13 @@ def dump_log(self, html=False, level='vdebug'):
             if record.levelno >= minlevel:
                 lines.append(fmt(record))
         return '\n'.join(lines)
-
+        
+    def change_log_capacity(self, capacity):
+        """
+        change log capacity according to user specifcation
+        """ 
+        ram_handler = RAMHandler(capacity=capacity)           
```

Comment:
Why are you creating a new (second) `RAMHandler` here? Is this just an oversight from deleting the previous code?